app/src/main/res/values/strings.xml

```diff
@@ -705,9 +706,9 @@
 
     <string name="filter_options">Options</string>
     <string name="filter_irreversible">Irreversible</string>
-    <string name="filter_irreversible_long">Irreversible. Drop instead of hide. Filtered toots will disappear irreversibly, even if filter is later removed.</string>
-    <string name="filter_word_match">Whole word</string>
-    <string name="filter_word_match_long">Whole word. Recommended to enable only phrases consisting of alphanumeric characters.</string>
+    <string name="filter_irreversible_long">Irreversible. Drop instead of hide. Filtered toots are lost forever, even if the filter is removed later.</string>
+    <string name="filter_word_match">The whole word</string>
```

Comment:
obey to Mastodon official.

Reply:
That is a marked noun error if you ask me. What seems to be the issue?